src/sql/platform/connection/common/connectionManagementService.ts

```diff
@@ -521,7 +521,7 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		if (!this.focusDashboard(connectionProfile)) {
 			let dashboardInput: DashboardInput = this._instantiationService ? this._instantiationService.createInstance(DashboardInput, connectionProfile) : undefined;
 			return dashboardInput.initializedPromise.then(() => {
-				this._editorService.openEditor(dashboardInput, { pinned: true }, ACTIVE_GROUP);
+				return this._editorService.openEditor(dashboardInput, { pinned: true }, ACTIVE_GROUP);
```

Comment:
The function is returning a Thenable, shouldn't it just be async?

Reply:
Certainly everywhere we use .then we could use async, but thats a larger change than what I'm looking to do here.